src/app/dataset/generalinfo/generalinfo.component.html

```diff
@@ -1,6 +1,9 @@
 <div class="metis-dataset-info-block" *ngIf="datasetData">
   <div class="dataset-name">
     <span class="svg-icon-dataset"></span> {{ datasetData.datasetName }}
+    <span *ngIf="isCurrentlyDepublished" class="mark-depublished">{{
+      'depublished' | translate
+    }}</span>
```

Comment:
Shouldn't this be a 3-way choice? 
1. Currently published
2. Currently depublished
3. Neither (has never been published).

Reply:
I thought we said it was reducible to a boolean?

Reply:
I thought we discussed a three way boolean (true, false or null)... but perhaps an enum is better.

Reply:
The design I see indicates that it either shows "depublished" or it doesn't.  I've made the underlying model an enum but reduced this to a boolean for this section of the UI